mrmustard/lab_dev/states/base.py

```diff
@@ -248,11 +252,36 @@ def fock_array(self, shape: Optional[Union[int, Sequence[int]]] = None) -> Compl
         """
         return to_fock(self.representation, shape).array
 
-    def phase_space(self) -> tuple[ComplexMatrix, ComplexVector]:
+    def phase_space(
+        self, s: float, characteristic: bool
+    ) -> tuple[complex, ComplexMatrix, ComplexVector]:
```

Comment:
```suggestion
    def phase_space(
        self, s: float
    ) -> tuple[ComplexMatrix, ComplexVector, complex]:
```

Reply:
if you need, modify the order which which the three outputs are returned (everywhere else we leave the coeff last not first)